README.md

```diff
@@ -257,6 +257,81 @@ When upgrading, IPFS may need to perform migrations and for that we need a stabl
 2. Make sure your firewall or antivirus is not blocking requests, such as P2P traffic;
 3. Try again, by restarting IPFS Desktop.
 
+### Error: Initializing daemon...
+
+These errors pop up from [ipfsd-ctl](https://github.com/ipfs/js-ipfsd-ctl) when the Kubo daemon fails to start up. Below are some scenarios where you may run into this error.
+
+#### Error: Your programs version (N) is lower than your repos (N+x).
+
+This means you are attempting to run an older version of ipfs-desktop or Kubo than you have previously ran on your machine. Each Kubo version (which is included with ipfs-desktop) is tied to a specific IPFS repo version, which you can see at https://github.com/ipfs/fs-repo-migrations#when-should-i-migrate.
+
+The ideal solution is to ensure you're running the latest version of ipfs-desktop, as upward migrations happen automatically. However, if you need to run the older version that is emitting this error, you will need to run a migration in reverse, manually. You can see the official instructions at https://github.com/ipfs/fs-repo-migrations/blob/master/run.md, but here is another set of instructions:
+
+1. Visit https://dist.ipfs.tech/#fs-repo-migrations and click the "download fs-repo-migrations" button, or select the appropriate version for your platform.
+1. extract the contents of the .tar.gz or .zip (your OS should be able to do this automatically) somewhere you can remember (such as ~/Downloads or c:/Users/myUserName/Downloads)
+1. open up a shell/terminal window, and run the following command:
+
+##### Linux/macOS/unix-ish platforms
+
+```bash
+cp -r ~/.ipfs ~/.ipfs.bak # backup current .ipfs folder. The path may be different, see [repository](#how-does-ipfs-desktop-select-the-ipfs-repo-location).
+cd ~/Downloads
+./fs-repo-migrations -revert-ok -to N # Where N is the current repo version
+```
+
+##### Windows (powershell)
+
+```powershell
+cp -r ~/.ipfs ~/.ipfs.bak # backup current .ipfs folder. The path may be different, see [repository](#how-does-ipfs-desktop-select-the-ipfs-repo-location).
+cd C:\Users\myUserName\Downloads
+./fs-repo-migrations.exe -revert-ok -to N # Where N is the current repo version
+
+```
```

Comment:
I think this is a power user option, last resort for data recovery. 
If someone wants to do this, they don't need this level of hand-holding.
Maybe trim to a single paragraph?

```suggestion
The ideal solution is to ensure you're running the latest version of ipfs-desktop, as upward migrations happen automatically. However, if you need to run the older version that is emitting this error, you will need to run a migration in reverse, manually. You can follow the official instructions [here](https://github.com/ipfs/fs-repo-migrations/blob/master/run.md) but with additional parameters: `fs-repo-migrations -revert-ok -to N`. See `fs-repo-migrations --help` for more information.
```

Reply:
Ack, thanks! Btw: if you want this to be useful to non-power users, you should include information how to disable autoupdate, otherwise the fix will be reverted on reboot.